projects/emissary-ingress/emissary/Makefile

```diff
@@ -0,0 +1,45 @@
+BASE_DIRECTORY:=$(shell git rev-parse --show-toplevel)
+GIT_TAG:=$(shell cat GIT_TAG)
+GOLANG_VERSION?="1.17"
+
+REPO=emissary
+REPO_OWNER=emissary-ingress
+
+BINARY_TARGET_FILES=emissary
+
+SOURCE_PATTERNS=./cmd/busyambassador
+
+EMISSARY_IMAGE_COMPONENT=emissary-ingress/emissary
+
+IMAGE_NAMES=emissary
+DOCKERFILE_FOLDER=./docker/linux/emissary-image
+
+CHART_VERSION=$(shell cat CHART_VERSION)
+
+HELM_PULL=true
+HELM_PULL_NAME=emissary-ingress
+HELM_PULL_LOCATION=datawire/emissary-ingress
+# IMAGE_REPO=public.ecr.aws/f5b7k4z5
+
+HAS_HELM_CHART=true
+HELM_DIRECTORY=charts/emissary-ingress
+HELM_DESTINATION_REPOSITORY=emissary-ingress/emissary
+HELM_IMAGE_LIST=emissary-ingress/emissary
+COPY_CRDS=$(BASE_DIRECTORY)/projects/emissary-ingress/emissary/helm/crds
+
+include $(BASE_DIRECTORY)/Common.mk
+
+BASE_IMAGE_NAME?=eks-distro-minimal-base-kind
+emissary/images/%: BASE_IMAGE_NAME=eks-distro-minimal-base-kind
```

Comment:
We wont want to use the kind base, its huge. Ill look closer at your dockerfile to better understand whats going on, but well need to work out a better base.

Reply:
Yea we can take a look, This seemed to be in many of the images
`remove_package bash`
Which was not going to work with the way Emissary runs since it's entrypoint is calling a bash script. If there is a smaller image that has bash though we can pivot easily.